packages/web/src/app/router.tsx

```diff
@@ -17,11 +27,22 @@ const router = createBrowserRouter([
           return redirect('/interlinear');
         },
       },
-      ...userRoutes,
+      {
+        path: 'admin',
+        element: <AdminView />,
+        loader: () => authorize('administer', 'User'),
+        children: [...userAdminRoutes, ...languageAdminRoutes],
+      },
+      ...userPageRoutes,
+      ...languagePageRoutes,
       ...translationRoutes,
-      ...languagesRoutes,
     ],
   },
+  {
```

Comment:
This is no comment on your code, just me trying to understand the router a little better...

How does this item not have to have the `path` field, but the first item (at line 19) does have a `path` field?

Is it because the first one needs to be able to render `/` AND its children, while this item is only for rendering its children routes?